src/open_inwoner/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -3581,52 +3712,115 @@ msgstr "Nevenvestiging"
 msgid "Rechtspersoon"
 msgstr "Rechtspersoon"
 
-#: kvk/models.py:12
+#: kvk/models.py:14 laposta/models.py:11
 msgid "API root"
 msgstr "API root"
 
-#: kvk/models.py:14
-msgid "The root of the API (e.g. https://api.kvk.nl/api/v1)"
-msgstr "De root van deze API (bijvoorbeeld https://api.kvk.nl/api/v1)"
+#: kvk/models.py:18
+msgid ""
+"The root of the API without version number and endpoint (e.g. https://api."
+"kvk.nl/api/ or https://api.kvk.nl/test/api)."
+msgstr "De root van de API zonder versienummer en endpoint (bijvoorbeeld https://api.kvk.nl/api/ of https://api.kvk.nl/test/api)"
 
-#: kvk/models.py:17
+#: kvk/models.py:23
 msgid "API key"
 msgstr "API key"
 
-#: kvk/models.py:20
+#: kvk/models.py:26
 msgid "The API key to connect to the KvK API."
 msgstr "De KVK API key om verbinding te maken met de KVK API."
 
-#: kvk/models.py:26
+#: kvk/models.py:32
 msgid "The SSL certificate file used for client identification. "
 msgstr ""
 "Het SSL-certificaatbestand dat wordt gebruikt voor clientidentificatie. "
 "Indien leeg gelaten, is wederzijdse TLS uitgeschakeld."
 
-#: kvk/models.py:34
+#: kvk/models.py:40
 msgid "The SSL/TLS certificate of the server."
 msgstr "Het SSL/TLS certificaat van de server"
 
-#: kvk/models.py:40
+#: kvk/models.py:46
 msgid "KvK configuration"
 msgstr "KVK configuratie"
 
 #: kvk/signals.py:24
 msgid "Retrieving data from KvK API based on KVK number"
 msgstr "Gegevens ophalen uit KVK API op basis van KVK-nummer"
 
-#: kvk/signals.py:32 kvk/tests/test_signals.py:144
+#: kvk/signals.py:32 kvk/tests/test_signals.py:137
 msgid "data was retrieved from KvK API"
 msgstr "data is opgehaald via KVK API"
 
-#: kvk/views.py:59 kvk/views.py:91
+#: kvk/tests/test_validators.py:12 kvk/validators.py:13
+msgid ""
+"The API root is incorrect. Please double-check that you didn't include the "
+"version number or endpoint."
+msgstr "De API-root is onjuist. Controleer nogmaals of het versienummer of het eindpunt niet zijn opgegeven."
+
+#: kvk/views.py:57 kvk/views.py:89
 msgid "Unauthorized"
 msgstr "Ongeautoriseerd"
 
-#: kvk/views.py:108
+#: kvk/views.py:106
 msgid "Invalid branch number for the current KvK number"
 msgstr "Ongeldig vestigingsnummer voor dit KVK-nummer"
 
+#: laposta/admin.py:48
+msgid "Advanced"
+msgstr "Geavanceerd"
+
+#: laposta/models.py:14
+msgid ""
+"The root of the API with version number (e.g. https://api.laposta.nl/v2/)."
+msgstr "De root van de API met versienummer (bijvoorbeeld https://api.laposta.nl/v2/)"
+
+#: laposta/models.py:18
+msgid "Basic Authentication username"
+msgstr "Basisverificatie gebruikersnaam"
```

Comment:
Zou eigenlijk 1 woord moeten zijn in NL, maar... is dit correct?

Reply:
Nee, hiermee wordt HTTP Basic Auth bedoeld, ik zou het als "Basic Authentication gebruikersnaam" vermelden om verwarring te voorkomen